### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/SFI-1694﻿

### CODE REVIEW

File: ffc-pay-batch-validator/retry.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { totalRetries } = require('./config')
+
+async function retry (fn, retriesLeft = totalRetries, interval = 1000, exponential = true) {
```
Comment (steve-dickinson) [inline ffc-pay-batch-validator/retry.js]: For consistency should this be an expression function instead of a declaration?

Comment (johnwatson484) [reply ffc-pay-batch-validator/retry.js]: Agreed.  Updated.

**Review by steve-dickinson — DISMISSED** [review]
LGTM

**Review by steve-dickinson — APPROVED** [review]
LGTM

**Pull request merged**